tests/test_events.py

```diff
@@ -51,58 +54,28 @@ def test_agenda_pdf_form_error():
         assert agenda_pdf_form.is_valid() == False
 
 
-def test_updates_made_true(event, event_document):
```

Comment:
I'd prefer not to lost tests for this functionality, if possible: https://github.com/datamade/la-metro-councilmatic/blob/6d2cc7532c4bcc4a7276338ccc2d0977c67cde5d/lametro/templatetags/lametro_extras.py#L130

Reply:
it's not possible. we'll deal with it this way https://github.com/datamade/la-metro-councilmatic/issues/509

Reply:
Whoops, missed your comment (https://github.com/datamade/la-metro-councilmatic/pull/516#pullrequestreview-328433632). Thanks.